tests/test_recipes.py

```diff
@@ -413,6 +413,23 @@ def test_increment_for_strings(self):
         person = baker.make_recipe("tests.generic.serial_person")
         assert person.name == "joe3"
 
+    def test_increment_for_strings_with_suffix(self):
```

Comment:
The tests are great! Would you mind adding a test where we have `start` and `suffix` together?

Reply:
Absolutely. I will update with another test.